.gitignore

```diff
@@ -3,5 +3,17 @@ dist/
 # Mac
 .DS_Store
 
+# env, cert, key, license
+.env*
+*.crt
+*.key
+*.license
+
+# Plugin
+*.tar.gz
+
+# node
+*.lock
+
 # Jetbrains
-.idea/
+.idea/
```

Comment:
Please add new line for end-of-line. Same comment for:
- e2e/playwright/.eslintignore
- e2e/playwright/.prettierignore
- e2e/playwright/setup-environment